init/builtins.cpp

```diff
@@ -380,7 +394,7 @@ static int do_mount(const std::vector<std::string>& args) {
 
 exit_success:
     return 0;
-
+#endif
```

Comment:
Since it spans across many lines I would add a comment:
#endif /* DISABLED_FOR_HYBRIS_SUPPORT */